web3/personal.py

```diff
@@ -1,62 +1,71 @@
-from web3.module import (
-    Module,
+from web3.method import (
+    Method,
 )
 
 
-class Personal(Module):
-    """
-    https://github.com/ethereum/go-ethereum/wiki/Management-APIs#personal
-    """
-    def importRawKey(self, private_key, passphrase):
-        return self.web3.manager.request_blocking(
-            "personal_importRawKey",
-            [private_key, passphrase],
-        )
-
-    def newAccount(self, password):
-        return self.web3.manager.request_blocking(
-            "personal_newAccount", [password],
-        )
-
-    @property
-    def listAccounts(self):
-        return self.web3.manager.request_blocking(
-            "personal_listAccounts", [],
-        )
-
-    def sendTransaction(self, transaction, passphrase):
-        return self.web3.manager.request_blocking(
-            "personal_sendTransaction",
-            [transaction, passphrase],
-        )
-
-    def lockAccount(self, account):
-        return self.web3.manager.request_blocking(
-            "personal_lockAccount",
-            [account],
-        )
-
-    def unlockAccount(self, account, passphrase, duration=None):
-        try:
-            return self.web3.manager.request_blocking(
-                "personal_unlockAccount",
-                [account, passphrase, duration],
-            )
-        except ValueError as err:
-            if "could not decrypt" in str(err):
-                # Hack to handle go-ethereum error response.
-                return False
-            else:
-                raise
-
-    def sign(self, message, signer, passphrase):
-        return self.web3.manager.request_blocking(
-            'personal_sign',
-            [message, signer, passphrase],
-        )
-
-    def ecRecover(self, message, signature):
-        return self.web3.manager.request_blocking(
-            'personal_ecRecover',
-            [message, signature],
-        )
+def default_root_munger(module, *args):
+    return [*args]
+
+
+def importRawKey():
+    return Method(
+        "personal_importRawKey",
+        mungers=[default_root_munger],
+        formatter_lookup_fn=None,
```

Comment:
Nice!

Seeing the new Method class used, I think I need to make `Method`'s arguments more intuitive.  I prefer omitting `formatter_lookup_fn` here rather than setting it to None.  Setting it to None looks like it would not have formatters applied, but it actually will have the default formatters applied.  I will make changes to `Method` to either remove `formatter_lookup_fn` from the arguments, or have it override the default formatters when set to None or a custom lookup function.

Reply:
Yeah, I would expect:
- "`formatter_lookup_fn` kwarg missing": default formatting
- "`formatter_lookup_fn=None`": no formatting
- "`formatter_lookup_fn=my_formatter`": only `my_formatter` runs